org.gnome.DejaDup.yaml

```
@@ -0,0 +1,62 @@
+# -*- Mode: YAML; indent-tabs-mode: nil; tab-width: 2 -*-
+---
+app-id: org.gnome.DejaDup
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+command: deja-dup
+
+finish-args:
+  - --filesystem=host
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.gtk.vfs.*
+
+modules:
+  - ./fasteners.json
```

```suggestion
  - fasteners.json
```

I think this is cleaner, same for the other.